core/textinput/src/textinput/TerminalConfigUnix.cpp

```diff
@@ -104,6 +106,11 @@ TerminalConfigUnix::HandleSignal(int signum) {
     }
   }
 
+  // gentle save and close if SIGTERM
+  if (signum == SIGTERM) {
```

Comment:
If I understood correctly, this would mean that after code similar to:
```
root [] TFile f(filename, RECREATE);
root [] do_stuff();
root [] if (all_is_well) f.Write();
```
We have a different behavior if the user does:
```
root [] .q
```
and
```
send SIGTERM
```
Where the former does not write the file while the later would now write the file.

Reply:
Yes.